pyaedt/hfss.py

```diff
@@ -5598,11 +5662,11 @@ def set_sbr_current_sources_options(self, conformance=False, thin_sources=False,
         self.logger.info("SBR+ current source options correctly applied.")
         return True
 
-    @pyaedt_function_handler()
+    @pyaedt_function_handler(positive_terminal="assignment", negative_terminal="reference")
     def set_differential_pair(
         self,
-        positive_terminal,
-        negative_terminal,
+        assignment,
+        reference,
         common_name=None,
         diff_name=None,
```

Comment:
I'd find something different for diff_name, commond_ref_z, diff_ref_z e matched @Samuelopez-ansys